gradle/libs.versions.toml

```diff
@@ -5,13 +5,13 @@ kotest = '4.6.4'
 testcontainers = '1.16.3'
 junit = '5.8.2'
 micronaut-serde = '1.0.1'
+micronaut-docs = '2.0.0'
 micronaut-gradle = '3.4.1'
-
 managed-pulsar = '2.10.0'
 
 [libraries]
 # Managed libraries
-managed-pulsar = { module = 'org.apache.pulsar:pulsar-client', version.ref= 'managed-pulsar' }
+managed-pulsar = { module = 'org.apache.pulsar:pulsar-client-original', version.ref= 'managed-pulsar' }

```

Comment:
Maybe we should change the alias name to `managed-pulsar-client` to make it clearer.